.rubocop.yml

```diff
@@ -74,10 +75,10 @@ Style/IndentationWidth:
 Style/IndentHash:
   Severity: fatal
   Enabled: true
+  EnforcedStyle: consistent
 
 Style/InlineComment:
-  Severity: fatal
-  Enabled: true
+  Enabled: false
```

Comment:
I would prefer comments to be allowed. There could be some complicated logic, we want to add a comment of why we're doing it.

E.g. checking GSSIDs for NI, or some calculation we're doing which isn't obvious.